packages/jest-worker/src/WorkerPool.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+'use strict';
+
+import BaseWorkerPool from './base/BaseWorkerPool';
+
+import type {
+  ChildMessage,
+  WorkerOptions,
+  OnStart,
+  OnEnd,
+  WorkerPoolInterface,
+  WorkerInterface,
+} from './types';
+
+export const canUseWorkerThreads = (): boolean => {
```

Comment:
any reason to put this behind a function rather than figuring it out during evaluation?

Reply:
readability - any reason not to?

Reply:
can be cached, no reason to evaluate it multiple times. Can be an IIFE

Reply:
fyi i reverted this because the it makes it easier to test (see breakage [here](https://circleci.com/gh/facebook/jest/40205?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)) and realistically it's only going to get called a few times when the pool is created